app/components/primer/beta/blankslate.rb

```diff
@@ -5,9 +5,11 @@ module Beta
     # Use `Blankslate` when there is a lack of content within a page or section. Use as placeholder to tell users why something isn't there.
     #
     # @accessibility
-    #   - Set the `heading` level based on what is appropriate for your page hierarchy. <%= link_to_heading_practices %>
-    #   - `secondary_action` can be set to provide more information that is relevant in the context of the `Blankslate`.
-    #   - `secondary_action` text should be meaningful out of context and clearly describe the destination. Avoid using vague text like, "Learn more" or "Click here".
+    #   - The blankslate uses a semantic heading that must be set at the appropriate level based on the hierarchy of the page.
+    #   - All blankslate visuals have been programmed as decorative images, using `aria-hidden=”true”` and `img alt=””`,  which will hide the visual from screen reader users.
+    #   - The blankslate supports a primary and secondary action. Both actions have been built as semantic links with primary and secondary styling.
+    #   - All links within the blankslate must receive visual and programmatic focus, and be accessible through mouse, keyboard and touch device.
```

Comment:
> All links within the blankslate must receive visual and programmatic focus 

Wondering if this is something the consumer needs to do. Does this happen by default if someone is setting a link using the slots: `c.secondary_action(href:`?

Reply:
@khiga8 per our convo, I will go ahead and remove this, as it's already taken care of in the component itself.